src/Media.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+
+const propTypes = {
+  children: PropTypes.element,
+  className: PropTypes.string,
+  left: PropTypes.element,
+  right: PropTypes.element,
+  tag: PropTypes.oneOfType([PropTypes.func, PropTypes.string]),
+};
+
+const defaultProps = {
+  tag: 'div',
+};
+
+const Media = (props) => {
+  const {
+    children,
+    className,
+    left,
+    right,
+    tag: Tag,
+    ...attributes,
+  } = props;
+
+  const classes = classNames(
+    className,
+    'media'
+  );
+
+  return (
+    <Tag {...attributes} className={classes}>
+      {left}
```

Comment:
This kind of goes against one of the goals for the project. "content is expected to be composed via props.children rather than using named props to pass in Components." Mostly for consistency. Any reason you didn't use a generic Media component for left/right. Placement could be determined by the order it is presented in the markup.